### ISSUE TITLE
Links to contribution guidelines are broken

### ISSUE THREAD

**Issue by marchof** [eclipse/jetty.project#11379]
**Jetty version(s)**
10.x - 12.x

**Jetty Environment**
n/a

**Java version/vendor** `(use: java -version)`
n/a

**OS type/version**
n/a

**Description**
The links to the contribution guidelines do not work:

```
Not Found

The requested URL /jetty/documentation/jetty-12/contribution-guide/index.html was not found on this server.

Additionally, a 404 Not Found error was encountered while trying to use an ErrorDocument to handle the request.
```

Looks like contribution guidelines are now hosted at this URL: https://eclipse.dev/jetty/documentation/contribution-guide/index.html

**How to reproduce?**

Follow the contribution guideline links on any of the following pages:

* https://github.com/jetty/jetty.project/blob/jetty-12.0.x/README.md
* https://eclipse.dev/jetty/documentation/jetty-10/
* https://eclipse.dev/jetty/documentation/jetty-11/
* https://eclipse.dev/jetty/documentation/jetty-12/

**Comment by joakime** [comment]
Fixed.

**Comment by marchof** [comment]
Hi @joakime, thanks for picking this up so quickly! The links on the documentation site are fixed now. But `README.md` still point to the non-exiting pages (two links).

**Comment by joakime** [comment]
Fixed in commits 2de4545ccb2473627c75e3620c859f8ce1e8b041 and b5e40d7e5b702b0140020e160efc876c001ed2cb
